<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>830</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CashEagle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionSecurity_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMinimize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCashEagle_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBorrower">
    <property name="title">
     <string>Clients</string>
    </property>
    <addaction name="actionApplicants"/>
    <addaction name="separator"/>
    <addaction name="action_Borrower"/>
    <addaction name="actionBorrowersList"/>
   </widget>
   <widget class="QMenu" name="menuLoan">
    <property name="title">
     <string>Loan</string>
    </property>
    <addaction name="action_Loans"/>
    <addaction name="actionLoan_Assess"/>
   </widget>
   <widget class="QMenu" name="menuPayment">
    <property name="title">
     <string>Payment</string>
    </property>
    <addaction name="actionMake_Payment"/>
    <addaction name="actionPayment_List"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <widget class="QMenu" name="menuBorrowers">
     <property name="title">
      <string>Borrowers</string>
     </property>
     <addaction name="actionList_All_Borrowers"/>
     <addaction name="actionNew_Applied"/>
    </widget>
    <widget class="QMenu" name="menuPayments">
     <property name="title">
      <string>Payments</string>
     </property>
     <addaction name="actionPayment_Due_Today"/>
     <addaction name="actionPayment_Made"/>
     <addaction name="actionPayment_Overdue"/>
    </widget>
    <widget class="QMenu" name="menuLoans">
     <property name="title">
      <string>Loans</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionNew_Loan"/>
     <addaction name="actionAging_Loan"/>
     <addaction name="actionCompleted_Loan"/>
     <addaction name="actionOverdue_Loan"/>
    </widget>
    <widget class="QMenu" name="menuCollection">
     <property name="title">
      <string>Collection</string>
     </property>
     <addaction name="actionCalls_Made"/>
    </widget>
    <addaction name="menuBorrowers"/>
    <addaction name="menuLoans"/>
    <addaction name="menuPayments"/>
    <addaction name="menuCollection"/>
   </widget>
   <widget class="QMenu" name="menuCollections">
    <property name="title">
     <string>Collections</string>
    </property>
    <addaction name="actionOverdue_List"/>
    <addaction name="actionCall_Logs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBorrower"/>
   <addaction name="menuLoan"/>
   <addaction name="menuPayment"/>
   <addaction name="menuCollections"/>
   <addaction name="menuReports"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Borrower"/>
   <addaction name="action_Loans"/>
   <addaction name="actionMake_Payment"/>
  </widget>
  <action name="actionCashEagle_Help">
   <property name="text">
    <string>CashEagle Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionSecurity">
   <property name="text">
    <string>Security</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSecurity_2">
   <property name="text">
    <string>Security</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Borrower">
   <property name="text">
    <string>Borrowers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionLocate_Borrowers">
   <property name="text">
    <string>Locate Borrowers</string>
   </property>
  </action>
  <action name="actionBorrowersList">
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="action_Loans">
   <property name="text">
    <string>Loans</string>
   </property>
  </action>
  <action name="actionLoan_Assess">
   <property name="text">
    <string>Loan Assess</string>
   </property>
  </action>
  <action name="actionCredit_Check">
   <property name="text">
    <string>Credit Check</string>
   </property>
  </action>
  <action name="actionMake_Payment">
   <property name="text">
    <string>Make Payment</string>
   </property>
  </action>
  <action name="actionPayment_List">
   <property name="text">
    <string>Payment List</string>
   </property>
  </action>
  <action name="actionList_All_Borrowers">
   <property name="text">
    <string>List All Borrowers</string>
   </property>
  </action>
  <action name="actionNew_Applied">
   <property name="text">
    <string>New Applied</string>
   </property>
  </action>
  <action name="actionNew_Loan">
   <property name="text">
    <string>New Loan</string>
   </property>
  </action>
  <action name="actionAging_Loan">
   <property name="text">
    <string>Aging Loan</string>
   </property>
  </action>
  <action name="actionCompleted_Loan">
   <property name="text">
    <string>Completed Loan</string>
   </property>
  </action>
  <action name="actionOverdue_Loan">
   <property name="text">
    <string>Overdue Loan</string>
   </property>
  </action>
  <action name="actionPayment_Due_Today">
   <property name="text">
    <string>Payment Due Today</string>
   </property>
  </action>
  <action name="actionPayment_Made">
   <property name="text">
    <string>Payment Made</string>
   </property>
  </action>
  <action name="actionPayment_Overdue">
   <property name="text">
    <string>Payment Overdue</string>
   </property>
  </action>
  <action name="actionCalls_Made">
   <property name="text">
    <string>Calls Made</string>
   </property>
  </action>
  <action name="actionOverdue_List">
   <property name="text">
    <string>Overdue List</string>
   </property>
  </action>
  <action name="actionCall_Logs">
   <property name="text">
    <string>Call Logs</string>
   </property>
  </action>
  <action name="actionApplicants">
   <property name="text">
    <string>Applicants</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
